.wuf/testkits/definitions.js

```diff
@@ -727,10 +721,7 @@ module.exports = {
   },
 
   AutoCompleteComposite: {
-    vanillaLegacyTestkit: true,
-    enzymeLegacyTestkit: true,
-    protractorLegacyTestkit: true,
-    noPuppeteerTestkit: true,
+    noPuppeteerTestkit: true, // was not exported before automation, keeping that way for now
```

Comment:
This fails the CI. 

It should be:
```suggestion
    vanillaLegacyTestkit: true,
    enzymeLegacyTestkit: true,
    protractorLegacyTestkit: true,
    noPuppeteerTestkit: true,
```